core/tools/compileCSSModules.js

```diff
@@ -26,11 +26,7 @@ const modules = [
   'date-picker',
   'drawer',
   'date-range-picker',
-  'editable-textfield',
   'overlay',
-  'footer',
-  'hero',
-  'icon',
```

Comment:
Those component folders do not have their own module.scss files to compile.

Reply:
I think that's a temporary issue in terms of these components.

Reply:
I see. As a remedy, would it be better to have that array list generated by searching over the `css/components` directory for only components with the module.scss, or better to keep the list there comprehensive but error handle when module.scss is missing?

Reply:
I agree that we should be searching for all of the module.scss files instead of using a defined list. If anything, the list should be the files to ignore.

Reply:
Per discussion with @pauljeter, We'll move forward with the list that is here, and I will make a new issue with notes on my efforts so far and come back to improve in another PR.